perf/istio/gateway-bouncer/setup.sh

```diff
@@ -0,0 +1,46 @@
+#/bin/bash
+set -ex
+
+NAMESPACE=${NAMESPACE:?"New/existing NAMESPACE to install this scenario into"}
```

Comment:
Most (maybe all) other tests set a default namespace. We should probably clean up the various scenarios to behave the same way - probably to have namespace configurable or have a default value?

Reply:
Yes, that's the idea, to be able to easily spin up this kind of test in isolation (where it makes sense). Namespace is currently a required param (similar to how DOMAIN is required in 'allconfig') as we definitely don't want to install this particular scenario into 'istio-system'.

Reply:
How about picking a default name, so someone running this doesn't have to worry about ? 
'gateway-bouncer' is as good name as any.